yolo_model:

  video_file:
    name: /home/hrjeong/kdarpa/tkDNN/demo/test2.mp4
  weights_file:
    name: /home/hrjeong/kdarpa/tkDNN/build/new_5_17_tiny_fp32.rt
  threshold:
    value: 0.5
  detection_classes:
    value: 2
  input_mode:
    value: 1 # if value=1 -> camera
            # else if value=0 -> video_file

